lib/services/concept_dictionary.dart

```diff
@@ -37,7 +41,7 @@ class ConceptDictionary {
     if (sessionId == null) {
       throw 'Authentication Failure';
     }
-    String url = '${AppUrls.omrs.concept}?q=$term&v=full';
+    String url = '${AppUrls.omrs.concept}?q=$term&v=custom:(uuid,name,display,conceptClass:(uuid,name)';
```

Comment:
why have you modified this method? this will break other features. Introduce a new method if you want a specific method to filter by class